lunatrace/bsl/hasura/migrations/lunatrace/1677516215552_vulnerable_reference_embeddings/up.sql

```diff
@@ -0,0 +1,56 @@
+CREATE EXTENSION vector;
+
+CREATE TABLE "vulnerability"."reference_content" (
```

Comment:
if this is a 1:1 with references, I'd usually say that it should just be part of the parent table. Thing is though, these columns are big, and while thats usually fine with a normal setup, im scared hasura might do some extremely stupid `SELECT * ...` queries. We should look into that